luaui/Widgets/gui_unit_stats.lua

```diff
@@ -811,19 +816,25 @@ local function drawStats(uDefID, uID)
 	DrawTextBuffer()
 end
 
+function enableStats()
+	showStats = true
```

Comment:
Just to make sure you are aware that when you don't return a truthy value you allow subsequent actions bound to this keybinding to be executed. Not an issue, just to let you know.

Reply:
I wasn't aware that it worked that way, thanks. I think allowing subsequent actions to be executed is a good thing because of the disableStats() action on releasing the key.